src/index.jsx

```diff
@@ -45,10 +45,10 @@ function LoginInfo() {
       </>
       {token ? (
         <>
-          <button onClick={() => logOut('rememberThis', idTokenData.session_state)}>Log out</button>
+          <button onClick={() => logOut('rememberThis', idTokenData?.session_state)}>Log out</button>
           <span style={{ margin: '0 10px' }}>
             Access token will expire at:{' '}
-            {new Date(localStorage.getItem('ROCP_tokenExpire') * 1000).toLocaleTimeString()}
+            {new Date(Number(localStorage.getItem('ROCP_tokenExpire')) * 1000).toLocaleTimeString()}
```

Comment:
`localStorage` actually only saves strings, so if we store a number, we should explicitly convert it to a number before doing arithmetics on it.